ui/components/app/wallet-overview/coin-overview.tsx

```diff
@@ -142,25 +229,99 @@ export const CoinOverview = ({
           <div className={`${classPrefix}-overview__balance`}>
             <div className={`${classPrefix}-overview__primary-container`}>
               {balanceToDisplay ? (
-                <UserPreferencedCurrencyDisplay
-                  style={{ display: 'contents' }}
-                  className={classnames(
-                    `${classPrefix}-overview__primary-balance`,
-                    {
-                      [`${classPrefix}-overview__cached-balance`]:
-                        balanceIsCached,
-                    },
-                  )}
-                  data-testid={`${classPrefix}-overview__primary-currency`}
-                  value={balanceToDisplay}
-                  type={PRIMARY}
-                  ethNumberOfDecimals={4}
-                  hideTitle
-                  shouldCheckShowNativeToken
-                  isAggregatedFiatOverviewBalance={
-                    !showNativeTokenAsMainBalance
-                  }
-                />
+                <>
+                  <Box onMouseEnter={handleMouseEnter} ref={setBoxRef}>
+                    <UserPreferencedCurrencyDisplay
+                      style={{ display: 'contents' }}
+                      className={classnames(
+                        `${classPrefix}-overview__primary-balance`,
+                        {
+                          [`${classPrefix}-overview__cached-balance`]:
+                            balanceIsCached,
+                        },
+                      )}
+                      data-testid={`${classPrefix}-overview__primary-currency`}
+                      value={balanceToDisplay}
+                      type={PRIMARY}
+                      ethNumberOfDecimals={4}
+                      hideTitle
+                      shouldCheckShowNativeToken
+                      isAggregatedFiatOverviewBalance={
+                        !showNativeTokenAsMainBalance && !isTestnet
+                      }
+                    />
+                  </Box>
+                  {shouldShowPopover &&
+                  !isTestnet &&
+                  !showNativeTokenAsMainBalance ? (
+                    <Popover
+                      referenceElement={referenceElement}
+                      isOpen={isOpen}
+                      position={PopoverPosition.BottomStart} // TODO check with design-team about this bottom start issue
```

Comment:
Hey @georgewrmarshall 👋 
We have these [designs](https://www.figma.com/design/aMYisczaJyEsYl1TYdcPUL/Portfolio-View?node-id=4098-126757&node-type=instance&focus-id=4186-130770&m=dev) show the arrow of popover at the very start of the component, even though i am using `PopoverPosition.BottomStart` , it still does not quite match the designs;
Did i maybe misused the component?
cc @amandaye0h 
<img width="1135" alt="Screenshot 2024-09-16 at 12 31 17" src="https://github.com/user-attachments/assets/c34e8d62-b061-485b-a634-37f286733073">

Reply:
The challenge here is that the Popover component has a lot of responsive behavior vs static design, but you could try `position=PopoverPosition.BottomStart` to see if that works.

Reply:
Hi @garrettbear ! Thnx for checking this out 🙏  The current code (which matches the screenshots) is already using `position={PopoverPosition.BottomStart} `
I was only able to modify the arrow position by doing what you see in video ☝️